Samples/MNIST/UWP/cs/DXCore_WinRTComponent/DXCore_WinRTComponent.vcxproj

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\Microsoft.Windows.CppWinRT.2.0.190506.1\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('..\packages\Microsoft.Windows.CppWinRT.2.0.190506.1\build\native\Microsoft.Windows.CppWinRT.props')" />
+  <PropertyGroup Label="Globals">
+    <CppWinRTEnabled>true</CppWinRTEnabled>
+    <MinimalCoreWin>true</MinimalCoreWin>
+    <ProjectGuid>{b3944f41-21ac-4d49-81a9-cb3f9b46f810}</ProjectGuid>
+    <ProjectName>DXCore_WinRTComponent</ProjectName>
+    <RootNamespace>DXCore_WinRTComponent</RootNamespace>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
+    <AppContainerApplication>true</AppContainerApplication>
+    <ApplicationType>Windows Store</ApplicationType>
+    <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
+    <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.18936.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformMinVersion>10.0.18925.0</WindowsTargetPlatformMinVersion>
```

Comment:
>18925 [](start = 42, length = 5)

18936? do you want to target latest insider SDK? ([https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewSDK](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewSDK)) latest is 18980

Reply:
Fixed (targeting latest SDK with "10.0")